cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -1,54 +1,51 @@
 import * as express from 'express';
 import * as passport from 'passport';
-import BaseUser from '../user/BaseUser';
-import UserSecService from '../user/UserSec';
+import {BaseUser} from '../user/BaseUser';
+import {UserSecService} from '../user/UserSec';
 
 interface Auth {
-  protect(req: express.Request, res: express.Response, next: () => void): void; // protect/authenticate?
-  login(opts?: object): void;
+  login(strategy: string, opts?: object): void;
+  protect(role: string, loginRoute: string): void;
```

Comment:
This is a param for the login route to redirect to if the user wasn't logged in yet. I was thinking of having some kind of config for this instead of being passed as a param?